### DISCUSSION TITLE
Presets stored in a subgroup

### DISCUSSION THREAD

**Discussion by siegenthalerroger** [renovatebot/renovate#18149] [Request Help] [answered]
Hi there,

I'm having issues with having a shared preset that's stored on gitlab in a nested subgroup. From what I can tell this is to the URL encoding not knowing that gitlab can even have nested subgroups... As an example

My `renovate.json` of the `elderbyte/lorem/ipsum` project contains the following:
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "gitlab>elderbyte/lorem/foo/renovate:elderSpringAngularStack.json5"
  ]
}
```

Which leads to the following error in the logs:
```
DEBUG: GitLab API 404 (repository=elderbyte/lorem/ipsum)
       "url": "https://gitlab.com/api/v4/projects/elderbyte%2Florem%2Ffoo%2Frenovate/repository/files/elderSpringAngularStack.json5/raw?ref=main"
DEBUG: Failed to retrieve elderSpringAngularStack.json5 from repo (repository=elderbyte/lorem/ipsum)
       "statusCode": 404,
       "url": "https://gitlab.com/api/v4/projects/elderbyte%2Florem%2Ffoo%2Frenovate/repository/files/elderSpringAngularStack.json5/raw?ref=main"
DEBUG: Preset fetch error (repository=elderbyte/lorem/ipsum)
```

Does Renovate support presets that are in nested sub-organisations (suborgs)?

**Comment by viceice** [comment, marked as answer]
use `//` instead of `:`

**Reply by siegenthalerroger** [reply]
Is this the new "correct" way, as the documentation still uses `:` in many places. I could give it a look around if that's the case.

**Reply by viceice** [reply]
the dot is for sub-preset in a json file. so yes use the newer syntax.
